packages/amplify-category-api/commands/api/add-graphql-datasource.js

```diff
@@ -205,3 +207,48 @@ async function getAwsClient(context, action) {
   const provider = require(providerPlugins[providerName]);
   return await provider.getConfiguredAWSClient(context, 'aurora-serverless', action);
 }
+
+function readSchema(projectDirectory, rdsSchema) {
+  const schemaFilePath = path.join(projectDirectory, 'schema.graphql');
+  const schemaDirectoryPath = path.join(projectDirectory, 'schema');
+  const schemaFileExists = fs.existsSync(schemaFilePath);
+  const schemaDirectoryExists = fs.existsSync(schemaDirectoryPath);
+  // convert the object schema to dump as file
+  const rdsSchemaDoc = mergeTypes([{}, rdsSchema], { all: true });
+
+  let schema;
+  if (schemaDirectoryExists) {
+    schema = readSchemaDocuments(schemaDirectoryPath).join('\n');
+    const rdsSchemaFilePath = path.join(schemaDirectoryPath, 'rds.graphql');
+    fs.writeFileSync(rdsSchemaFilePath, rdsSchemaDoc, 'utf8');
+  } else if (schemaFileExists) {
+    schema = fs.readFileSync(schemaFilePath).toString();
+    const rdsSchemaFilePath = path.join(projectDirectory, 'rds.graphql');
+    fs.writeFileSync(rdsSchemaFilePath, rdsSchemaDoc, 'utf8');
```

Comment:
When there is no `schema` directory transformer uses only `scheam.graphql` and it wont read `rds.graphql`. You should merge the documents and save it as schema.graphql

Reply:
When there is no schema directory, I am just returning normal schema.graphql in readSchema function and merging of schema is happening after that.